uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceType.tpl

```diff
@@ -1,13 +1,11 @@
-{if $action eq 1 or $action eq 2}
+{if in_array($action, array(1,2))}
```

Comment:
Can we use `[1, 2]` notation instead of `array(1, 2)`?

Reply:
I don't think we can. Using `[]` as `{if in_array($action, [1,2])}` led to error
```
User error: Smarty error: [in CRM/HRLeaveAndAbsences/Page/AbsenceType.tpl line 1]: syntax error: unbalanced parenthesis in if statement (Smarty_Compiler.class.php, line 1277) in Smarty->trigger_error() (line 1100 of /vagrant/test2/sites/all/modules/civicrm/packages/Smarty/Smarty.class.php).
```